pyext/src/rmsd_calculation.py

```diff
@@ -325,6 +348,13 @@ def get_rmfs_coordinates_one_rmf(path, rmf_A, rmf_B, subunit_name=None,
                             residue_in_bead + "_" + str(copy_number))
             mod_id += 1
 
+    if symm_groups_file:
+        for grp in symm_groups:
+            if len(grp) == 0:
+                print("Warning. Symmetry option specified but created "
```

Comment:
Using Python's `warnings` module here would be more flexible (e.g. warnings can be turned off or converted to errors by the user, and we can check for them in test cases). But it's a bit of a nuisance to set up.